WTWSMS/history/titles/b_ostia.txt

```diff
@@ -559,7 +559,7 @@
 	holder = 0
 }
 431.1.1 = {
-	holder = 1006016 # Auchenius Bassus (II)
+	holder = 1006016 # Auchenius Bassus (III)
```

Comment:
Wasn't he the second consul named Auchenius Bassus?

Reply:
See above.